File: OMS.services/OMS.StaffService/OMS.StaffService/Models/StaffModel.cs
using System.ComponentModel.DataAnnotations;

namespace OMS.StaffService.Models
{
    public class StaffModel
    {
        [Key]
        public Guid Staff_Id { get; set; }



        [MaxLength(25)]
        public string First_Name { get; set; }
        [MaxLength(25)]
        public string Last_Name { get; set; }
        [MaxLength(25)]
        public string Middle_Name { get; set; }
        [DataType(DataType.EmailAddress)]
        public string Email { get; set; }
        public string Cell_Phone { get; set; }
        public string Relationship { get; set; }
        [MaxLength(250)]
        public string Address { get; set; }
        public string State { get; set; }
        public long NIN { get; set; }    
        [MaxLength(7)]   
        public string Work_ID { get; set; }
        public string Sex { get; set; }
        public string DOB { get; set; }



        public DateTime Created { get; set; }
        public DateTime? Updated { get; set; }
    }
}

File: OMS.services/OMS.StaffService2/models/staffModel.js
import  { Schema, model } from "mongoose";




const staffSchema = new Schema({
    Email: {
        unique: true,
        required: true,
        type: String
    },
    First_Name: {
        required: true,
        type: String
    },
    Last_Name: {
        required: true,
        type: String
    },
    Middle_Name: {
        required: true,
        type: String
    },
    Address: {
        required: true,
        type: String
    },
    Cell_Phone: {
        unique: true,
        required: true,
        type: String
    }, 
    DOB: {
        required: true,
        type: String
    }, 
    NIN: {
        unique: true,
        required: true,
        type: String
    }, 
    Profile_Url: {
        type: String
    }, 
    Relationship: {
        required: true,
        type: String
    }, 
    Sex: {
        required: true,
        type: String
    }, 
    State: {
        unique: true,
        required: true,
        type: String
    },
    Work_ID: {
        unique: true,
        required: true,
        type: String
    }
}, { timestamps: true });

const Staff = model("Staff", staffSchema);

export default Staff;